src/providers/data_providers/yelpBusinessImageProvider.ts

```diff
@@ -0,0 +1,21 @@
+import { OpenCageApiUtils } from '../../utils/openCageApiUtils';
+import { YelpFusionApiUtils } from '../../utils/yelpFusionApiUtils';
+import { BusinessInfoInput } from '../../models/data_models/types';
+import { BaseBusinessImageProvider } from './baseBusinessImageProvider';
+import { Image } from '../../models/data_models/image';
+
+export class YelpBusinessImageProvider extends BaseBusinessImageProvider {
+	protected imageProviderName: string = 'yelp';
+
+	public async getBusinessImages(businessInfo: BusinessInfoInput): Promise<Array<Image>> {
+		const yelpApi = await new YelpFusionApiUtils(this.config).init();
+		const geocode = await new OpenCageApiUtils(this.config)
```

Comment:
As discussed with @tim-zwart in the Discord, we could try to create a public static async `.from(config)` factory method for the class, and then make the constructor private. The `from` method would perform the logic of both the constructor and `init`. This would prevent anyone from accidentally calling `new` without `.init`.